### ISSUE TITLE
Adding Issue/Feature request submitting Template

### ISSUE THREAD

**Issue by helios1101** [jainaman224/Algo_Ds_Notes#722]
I'll like to work on this issue @jainaman224

**Comment by helios1101** [comment]
This is my template if you can review it once. 


### I'm submitting a :
 - [ ] feature request
 - [ ] bug request
- - - 
### Description :
* Any description is support of your request.
- - - 
### Current behaviour:
* Currently how it's behaving
- - - 
### Expected behaviour:
* After your request goes approves what will be the behaviour
- - - 
### Would you like to work on the issue, or should we assign someone else?
 - [ ] Yes, I'll work
 - [ ] Assign someone else

**Comment by jainaman224** [comment]
Please remind me on Sunday.

**Comment by helios1101** [comment]
@jainaman224 

> Please remind me on Sunday.

**Comment by jainaman224** [comment]
Let's first think about PR template. I am thinking and reading about ways to organise issues. Something like master issue and then sub issues.

**Comment by jainaman224** [comment]
Once I get clarity, we will discuss. Thanks a lot for reminding.

**Comment by helios1101** [comment]
@khushal87 This was an issue which @jainaman224 was talking about.

**Comment by khushal87** [comment]
Okay

**Comment by jainaman224** [comment]
@khushal87 Any inputs?

**Comment by khushal87** [comment]
Like the template would be more easier than the one which you uploaded if it would had predefined language selector according to my opinion

**Comment by jainaman224** [comment]
But there so many languages. So, it will becomes difficult which ones are selected.

**Comment by khushal87** [comment]
Yeah . But there can be standard languages like c,c++,java,javascript etc and then a section to mention other languages if not in checkbox @jainaman224

**Comment by jainaman224** [comment]
For the time being let this to be this way.

**Comment by khushal87** [comment]
Okay @jainaman224
